examples/crds/ciliumnodes.yaml

```diff
@@ -253,6 +253,11 @@ spec:
                     items:
                       description: AzureInterface represents an Azure Interface
                       properties:
+                        GatewayIP:
+                          description: "GatewayIP is the interface subnet's default
```

Comment:
Had to read this a few times; did you mean `the interface's subnet's default route`?

Reply:
Fixed.